docs/reference/metricbeat/metricbeat-module-gcp.md

```diff
@@ -361,6 +361,16 @@ metricbeat.modules:
   exclude_labels: false
   period: 1m
   collect_dataproc_user_labels: true
+
+- module: gcp
+  metricsets:
+    - vertexai_logs
+  period: 300s  # 5 minutes
+  project_id: "your-project-id"
+  table_id: "your-project-id.dataset.id.table_name"
```

Comment:
why do we make the customer specify the project ID twice? once in the `project_id` field, and once in the `table_id` field

Reply:
Correct we can scrape it out of table id. 
Although table id is not to be formed by customer. Its readily available to be copied from TableID under scehema as mentioned in the documentation [here](https://github.com/elastic/beats/pull/46383/files#diff-4c6812e7c6fb39e96e4f0be75316adc55794fa625a58d8d84bdf93c438d74116R26).